src/generator/AutoRest.Go/Extensions.cs

```diff
@@ -53,10 +53,32 @@ public static string ToSentence(this string value)
             else
             {
                 value = value.Trim();
+                if (value.StartsWithAcronym())
+                {
+                    return value;
+                }                
                 return value.First().ToString().ToLowerInvariant() + (value.Length > 1 ? value.Substring(1) : "");
             }
         }
 
+        public static bool StartsWithAcronym(this string value)
```

Comment:
Can you please add a comment explaining the algorithm this function uses?  It looks like it iterates over the string and if it's all caps then it returns true (seems reasonable however it does depend on the swagger properly casing them).